uint16 id
float32 rvec1
float32 rvec2
float32 rvec3
float32 tvec1
float32 tvec2
float32 tvec3